algorithms/linfa-ensemble/src/ensemble.rs

```diff
@@ -66,8 +68,8 @@ where
         let mut y_array = y.as_targets_mut();
         assert_eq!(
             x.nrows(),
-            y_array.len(),
-            "The number of data points must match the number of output targets."
+            y_array.len_of(Axis(0)),
```

Comment:
change to `nrows`

Reply:
This doesn't seem to be possible since I've written this to be as general as possible in the dimensionality of y. As far as I can tell `nrows` is only defined for 2d arrays -- `ArrayBase<S, Ix2>` -- while `y_array` here has type `ArrayBase<ViewRepr<&mut <T as AsTargets>::Elem>, <T as AsTargetsMut>::Ix>`

Reply:
OK makes sense